pandas/tests/apply/test_series_apply.py

```diff
@@ -364,18 +369,32 @@ def test_with_nested_series(datetime_series):
 def test_replicate_describe(string_series):
     # this also tests a result set that is all scalars
     expected = string_series.describe()
-    result = string_series.apply(
-        {
-            "count": "count",
-            "mean": "mean",
-            "std": "std",
-            "min": "min",
-            "25%": lambda x: x.quantile(0.25),
-            "50%": "median",
-            "75%": lambda x: x.quantile(0.75),
-            "max": "max",
-        }
-    )
+    if get_option("new_udf_methods"):
+        result = string_series.agg(
```

Comment:
Changed from apply to agg as now apply with a dictionary of ops will act the same way as apply with a single op - acting on each row of the Series individually.